### DISCUSSION TITLE
Accessibility Features (Reduce Motion)

### DISCUSSION THREAD

**Discussion by goodnightmonkey** [stoatchat/discussions#561] [Feature Suggestions]
Accessibility features like Reduce Motion (with options to disallow motion completely or permit them to animate when hovering over them) would be great. I personally get super motion sickness, so I've been struggling with the animated emoji and gifs.

I'm not sure what others would apply. High contrast is already resolved with one being able to make or gran a high contrast theme (loving this feature, by the way!).

Thanks for reading.
(Note: I only checked the first couple of pages for this and didn't see it. Apologies if it is already out there.)

**Comment by deli73** [comment]
very much seconded! we have trouble focusing on anything when constant motion is on screen so we could benefit greatly from this too; there's a lot of reasons for reduced motion options to be a thing

**Comment by MasicoreLord** [comment]
Here's a work-around for now: (put this in your custom css in the Appearance tab)
```css
* {
    animation: none!important;
    transition: none!important
}
```

**Comment by SympathyTea** [comment]
Absolutely this, along with perhaps color blind options? Alt text or text descriptions would also be great as well for screen reader users.

**Comment by prismflasher** [comment]
Disabling interface animations with custom css seems to work great (though less tech savvy users won't even think of it being possible), but I'd really like to see the ability to disable animated emotes/stickers/icons/etc. or even gifs sent in chat. For me, this is the most important feature discord has over its competitors (especially as many of them, revolt included, allow all users to upload and use animated images in most places images can be uploaded, causing them to be significantly more common), and I'd really hate to be stuck on one platform over accessibility features.

**Reply by EyesEyesBaby** [reply]
Agreed, the ability to freeze animations from emotes and gifs (making them play by clicking/hovering over them) is important enough that I personally know at least three people where that's one of the most important functions for a chat client (one due to hallucinatory issues, another due to attention issues, the third due to motion sickness issues.)

**Comment by without-ordinary** [comment]
This is a basic accessibility requirement.

I cannot look at any of my group chats without getting nauseous from the occasional animated thing that may happen to be in chat history.

**Reply by without-ordinary** [reply]
The plan to move our community over to Stoke was canceled. This and many other accessibility issues were some of the key reasons for that decision.

**Comment by Cerespirin** [comment]
+1 from me. The server discovery feature is blinding to the point of being useless for me right now.

**Comment by lobster-wing** [comment]
I only want to remove the message-wide highlight that happens whenever your mouse pointer hovers over any message (on discord this was part of reduced motion for some reason), is the custom css option still somewhere? I can't seem to find it

**Comment by Triffen** [comment]
This! I also get motion sickness and migraines from motion in my side vision, and love Discord's accessibility features that allow them to only play on hover.

**Comment by lyalkerose** [comment]
This is a basic accessibility requirement that should have already been implemented yesterday (figure of speech). Not having this accessibility option makes Stoat unusable and actually unsafe for those with photosensitivity, seizure disorders and epilepsy. Stoat devs, you should research SUDEP (sudden unexpected death in epilepsy.) Someone abusing your platform with the lack of ability to turn off GIF autoplay to deliberately trigger a seizure can absolutely implicate you in an assault case.

**Reply by MaguMogu123** [reply]
^ entirely, agreed
